### PR TITLE
docs: traceability kit and insdustry core kit for release 24.03

### PR DESCRIPTION
**Pull request by mhellmeier**
## Description
This pull request introduces an updated version 4.0.0 of the Traceability KIT (fixes https://github.com/eclipse-tractusx/sig-release/issues/530) and an initial version of the Industry Core KIT in version 1.0.0 (fixes https://github.com/eclipse-tractusx/sig-release/issues/532), compatible with the overall 24.03 release.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Traceability Kit/page_business_view.mdx [hunk 1]
```diff
@@ -9,67 +9,90 @@ sidebar_position: 1
...
+- SPDX-FileCopyrightText: 2023 - 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023 - 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
+- SPDX-FileCopyrightText: 2023 - 2024 German Edge Cloud GmbH & Co. KG
+- SPDX-FileCopyrightText: 2023 - 2024 Mercedes Benz AG
+- SPDX-FileCopyrightText: 2023 - 2024 Robert Bosch Manufacturing Solutions GmbH
+- SPDX-FileCopyrightText: 2023 - 2024 SAP SE
+- SPDX-FileCopyrightText: 2023 - 2024 Siemens AG
+- SPDX-FileCopyrightText: 2023 - 2024 T-Systems International GmbH
+- SPDX-FileCopyrightText: 2023 - 2024 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2023 - 2024 Contributors to the Eclipse Foundation
```
Comment (maximilianong) [inline docs-kits/kits/Traceability Kit/page_business_view.mdx:22]: Is it 2023 - 2024 or 2023, 2024 
@AngelikaWittek can you give guidance here - just to have it the same way on all pages, thanks.

**Review by maximilianong — DISMISSED** [review]
Good update!
Thanks @mhellmeier and team.
Ready for release 24.03.

**Comment by maximilianong** [comment]
@mhellmeier 
Sorry - we did not saw that this is one PR for Traceability and Industry Core.
First review was just for Traceability.

**Review for Industry Core KIT:**

- [ ] The sidebar navigation of KITs is structured A-Z, I see Industry Core now on top, can you please adjust this accordingly.
- [ ] Changelog needs to be adapted to version 0.1.0 with a first release.
- [ ] Notice can just start with 2024
- [ ] Standards : can you list / name the relevant standards and their numbers + version, even if you cannot link to them because they are not published so far
- [ ] Operations View: is the SDE still valid as a reference implementation?

File: docs-kits/kits/Traceability Kit/page_business_view.mdx (hunk 1 above)
Comment (mhellmeier) [reply docs-kits/kits/Traceability Kit/page_business_view.mdx:22]: Any guidance here, @AngelikaWittek ? The [Eclipse Committer Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) mentions to only use the "year in which the content was created", while the current [Open Source TRG 7.02](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) mentions a list of years.

Which approach should we follow?

**Comment by maximilianong** [comment]
@mhellmeier 
Friendly reminder - Please fix the requested changes by 04.03.2024 (EOB) so that we can perform the final checks and prepare the release communication. Thanks!

File: docs-kits/kits/Traceability Kit/page_business_view.mdx (hunk 1 above)
Comment (danielmiehle) [reply docs-kits/kits/Traceability Kit/page_business_view.mdx:22]: After consultation with @AngelikaWittek , please use the handbook guidelines. At the same time, we are in the process of adapting the TRG accordingly. This change refers to the representation of the year.

Comment (mhellmeier) [reply docs-kits/kits/Traceability Kit/page_business_view.mdx]: Okay, I just changed the copyright year to 2023 (creation year) as mentioned in the Eclipse Committer Handbook.

**Comment by danielmiehle** [comment]
> Both arguments are correct in their way. There are two options (both are fine for me).
> 
> 1. We mention the SDE in the Operation View
> 2. We remove entirely the Operation View page from the Industry Core

Thanks for your feedback.
Let's go with option 1!

**Comment by mhellmeier** [comment]
> Thanks for your feedback. Let's go with option 1!

Okay, great. Option 1 is still implemented and documented in the current version. Therefore, and since all issues and open points are addressed, this pull request is ready to be merged, @maximilianong + @danielmiehle!

**Review by maximilianong — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Pull request merged**